docs/source/user-guide/general/deploying_on_ec2.rst

```diff
@@ -0,0 +1,11 @@
+====================
+Deploying on AWS EC2
```

Comment:
I would add the following paragraph:
Installing our open source solution on your AWS servers allows you to enjoy a reliable solution, limited to 1 model, with Amazon grade security that is accessible to the relevant teams in your organization. Our minimal configuration includes a secure environment. 
For a more scalable installation, you should use our on-premise version, which allows you to use more than 1 model , have integration with your company's authentication and authorization services and several other features that are not available in the open-source distribution. reach out to us at : customersuccess@deepchecks.com(mailto:customersuccess@deepchecks.com)